src/main/scala/firrtl/proto/FromProto.scala

```diff
@@ -8,7 +8,7 @@ import java.io.{File, FileInputStream, InputStream}
 import collection.JavaConverters._
 import FirrtlProtos._
 import com.google.protobuf.CodedInputStream
-import Firrtl.Statement.ReadUnderWrite
+import Firrtl.Statement.{ReadUnderWrite, Formal}
```

Comment:
`Formal` should be `Verification`

Reply:
? There is already a `Firrtl.Statement.Verification`. Verification is used to denote the statement and formal is used to denote the "operator" (assert, assume, cover)

Reply:
I see. The name `Formal` is a little confusing, but I remember us talking about this on the Zoom call and we only came up with thing we did not want to call it like `Kind` and `Type`.

Reply:
The only directly analogous situation is DoPrim, where the field for the “variety” of operator is just called `op`

Reply:
So `VerificationOp` ?

Reply:
> So `VerificationOp` ?

Eh, it's not evocative but I guess I could live with it. But think of how this Formal enum is actually used in the code, like:
```
case Verification(Formal.Assert, ...) =>
```
IMO it reads fine and it's obvious what is meant. Writing that as `case Verification(VerificationOp.Assert, ...)` feels pretty bloodless.

Reply:
Not a blocker. I don't care that much.